neutrons/reflectivity_ui
# Documentation

### Description of work:


### To Test by Reviewer:
```bash
cd docs/
make html
```
This should produce the documentation under `/docs/_build/html`

The reviewer should check for the discussed software practices

- [ ] all internal functions should have an underbar, as is python standard
- [ ] need more comments in meat of code explaining intent
- [ ] does the code follow the other [coding standards](https://scse.ornl.gov/drtsans/developer/index.html)?
